src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,6 +5,8 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int tonnage;
+
```

Comment:
if you've added a filed, you must add a constructor, cause without it, the field becomes unreachable